seismic-app/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+
+  seismic-app:
+    container_name: seismic-app
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - '.:/app'
+      - '/app/node_modules'
```

Comment:
@tudoramariei Just noticed this... It's OK to mount `node_modules` as a volume, especially when using Docker for Mac or Windows, but this means that you'll not have the node_modules folder on your machine, so you are loosing IDE support for those libraries. There is a way to configure this and have node_modules synced with the host machine, but it will incur a performance penalty on Windows (smalish) or MacOS (bigger).
So the questions is what do we want: IDE support for libraries or a bit more performance?

Reply:
Most likely we're going to go for IDE support for the development part
It's also probably going to be useful to set multiple Dockerfiles `docker-compose` for deploying to production, I guess that's where I'd like to keep that line

Reply:
I'm in the same boat with IDE support. In production, volumes should be used only for data, not code.